manager/testdata/notebook/README.md

```diff
@@ -0,0 +1,347 @@
+# Notebook sample to demonstrate write flow
+
+This sample shows how Fybrik enables a Jupyter notebook workload to read and write a dataset: a data that resides in an object storage is first been read into the notebook dataset and then written into a new location in the object storage. It is assumed that before the fybrik application is launched the new location already exists in the object store and the catalog holds the information about the new written asset.
+This sample demonstrates how policies are seamlessly applied when accessing the dataset classified as financial data.
+
+In this sample you play multiple roles:
+
+1. As a data owner you upload a dataset and register it in a data catalog
+2. As a data steward you setup data governance policies
+3. As a data user you specify your data usage requirements and use a notebook to consume the data
+
+## Before you begin
+
+- Install Fybrik using the [Quick Start](../get-started/quickstart.md) guide.
+  This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
+- A web browser.
+
+## Create a namespace for the sample
```

Comment:
I would suggest separating what needs to be done for the read flow and what needs to be done for the write flow to make it clearer.